jobs/JEVS_AQM_PREP

```diff
@@ -52,28 +61,18 @@ export MASKS=${MASKS:-$FIXevs/masks}
 
 echo $COMPATH
 
-export NET=${NET:-evs}
-export STEP=${STEP:-prep}
-export COMPONENT=${COMPONENT:-aqm}
-export RUN=${RUN:-atmos}
-export VERIF_CASE=${VERIF_CASE:-grid2obs}
-export MODELNAME=${MODELNAME:-aqm}
-export modsys=${modsys:-aqm}
-export mod_ver=${mod_ver:-${aqm_ver}}
-
-
 #
 #  The AirNOW obs in DCOM will continue to be updated after first
 #  created upto 48 hours.
 #  Process the observation input only when it stop updating.
 #
-export VDATE=${VDATE:-${PDYm3}}
+export INITDATE=${INITDATE:-${PDYm3}}
```

Comment:
I noticed both JEVS_AQM_STATS and JEVS_AQM_PLOTS both set VDATE to PDYm3. What is the purpose for changing VDATE to INITDATE in the prep jobcard?

Reply:
This is the new EVS standard and is required to be fixed as specified in AQM elements in [spreadsheet](https://docs.google.com/document/d/1vOGnCwCM3COUhQlTqHyBZrYSCPx9b6AYnxLc5NtVGEk/edit) (under AQM)

Reply:
@AndrewBenjamin-NOAA @Ho-ChunHuang-NOAA To provide some additional context:

EVS prep steps gather forecasts by their INITDATE (i.e., Initialization Date) and stats/plots jobs do verification based on the VDATE (i.e., Valid Date). As an example, a prep step might grab the AQM forecasts from the 00Z, 06Z, 12Z, and 18Z initialization times. The prep step does **not** grab forecast files across 20+ initialization times that are all valid at the same VDATE. If we did things that way, forecast files may have rolled off of /dcom and /com by the time we needed them. It's better to grab an entire 12Z cycle at once, and before things roll off, so INITDATE is used in the prep step. I hope this helps!

Reply:
@AliciaBentley-NOAA I have not problem with the new requirement, but I want to note that AQM prep processed both OBS (by VDATE) and MOD data (by INITDATE).  This maybe different from other components.